src/components/createBook/BookForm.tsx

```diff
@@ -55,15 +95,49 @@ function CreateBook() {
   const handleSelectedAuthorsChange = (authorsData: MultiValue<AuthorResponse>) => {
     setSelectedAuthors(authorsData as AuthorResponse[])
   }
+  const handleBookFormSubmit = () => {
+    if(bookId) {
+      updateBook()
+      return
+    }
+    createBook()
+  }
+
+  const updateBook = () => {
+    if(validateInput()) {
+      bookService.update(prepareUpdateFormData())
+        .then(() => {
+          navigate('/Books')
+        })
+        .catch(error => {console.error(error)})
```

Comment:
console.error doesn't do much from users point of view. You should replace this in future with proper notification message. You can use something like notistack, toastify or do it custom.